### PR TITLE
Fix cb pk

### LINKED ISSUE #99: 🐛 CB model does not include cb id & org id
`community_business_id`'s & `organisation_id`'s are different, however both aren't included in the model for community businesses. 

`organisation_id` is needed to to link users to roles.

### PR DESCRIPTION
**Pull request by astroash**
- removes primary keys for community_business table and funding_body table
- updates CommunityBusiness model to not use cb_id
- updates online schema to match

fixes #99

### CODE REVIEW

File: src/models/__tests__/community_business.test.integration.ts [hunk 1]
```diff
@@ -35,7 +35,7 @@ describe('Community Business Model', () => {
     });
 
     test('get :: filter orgs by ID | non-existent ID resolves to empty array', async () => {
-      const orgs = await CommunityBusinesses.get(knex, { where: { id: 200 } });
+      const orgs = await CommunityBusinesses.get(knex, { where: { organisationId: 200 } });
```
Comment (eliascodes) [inline src/models/__tests__/community_business.test.integration.ts]: We can still unambiguously call this `id`. `organisationId` is verbose and adds no value. Remember, the point of the models is to present a convenient API that is decoupled from the database implementation.

File: database/migrations/sql/20180604_08_create_visits.sql [hunk 2]
```diff
@@ -27,7 +27,7 @@ CREATE TABLE visit_activity (
   deleted_at                 TIMESTAMP WITH TIME ZONE,
 
   CONSTRAINT visit_activity_pk                            PRIMARY KEY (visit_activity_id),
-  CONSTRAINT visit_activity_to_community_business_fk      FOREIGN KEY (community_business_id)      REFERENCES community_business ON DELETE CASCADE,
+  CONSTRAINT visit_activity_to_community_business_fk      FOREIGN KEY (organisation_id)            REFERENCES organisation ON DELETE CASCADE,
```
[the remark this answers is not in the document]
Comment (astroash) [reply database/migrations/sql/20180604_08_create_visits.sql]: I left the reference on the online scheme to not mess up the lines. On our actual db it made sense to point to the table where the key originated from.

File: database/migrations/sql/20180604_04_create_organisations.sql [hunk 3]
```diff
@@ -80,14 +76,14 @@ CREATE TABLE community_business (
 
 CREATE TABLE funding_programme_community_business (
   funding_programme_id  INT NOT NULL,
-  community_business_id INT NOT NULL,
+  organisation_id       INT NOT NULL,
```
[the remark this answers is not in the document]
Comment (astroash) [reply database/migrations/sql/20180604_04_create_organisations.sql]: Not sure what you mean. `community_business_id` no longer exists.

**Comment by eliascodes** [comment]
Also, I've added `organisation_type` back to the `organisation` table in the schema diagram. My thinking is that now that everything references `organisation` instead of either `community_business` or `funding_body`, we may want to save ourselves a join and be able to do something like:
```sql
SELECT * FROM foo 
INNER JOIN organisation ON organisation_id 
WHERE organisation.organisation_type = 'CB';
```
instead of
```sql
SELECT * FROM foo 
INNER JOIN organisation ON organisation_id 
INNER JOIN community_business ON organisation_id;
```

Please could you also add this extra column into the schema, with an `ENUM` data type, with values `community business` and `funding body`.

**Comment by eliascodes** [comment]
@astroash did we settle on dropping the `organisation_type` column after all? It's still in the diagram.

**Comment by eliascodes** [comment]
@astroash actually, nevermind, I also have some changes to make, I'll update it.

**Pull request merged**